### PR TITLE
NP-2401 Add cluster uninstall and decomission

### PR DESCRIPTION
**Pull request by dhiguero**
#### What does this PR do?

* Add two new commands to uninstall and decommission application clusters.

#### What is missing?

* Overall command refactoring will be done in other PR

#### Any background context you want to provide?

Two new fronts are opened in this PR. First, argument renaming in the command help as arguments that are mandatory should be shown as `<arg>`. And secondly, introduction of command help with per command examples.

#### What are the relevant tickets?

- [NP-2401](https://nalej.atlassian.net/browse/NP-2401)

### CODE REVIEW

File: internal/pkg/server/clusters/handler.go [hunk 1]
```diff
@@ -73,7 +73,9 @@ func (h *Handler) ProvisionAndInstall(ctx context.Context, request *grpc_provisi
 
 // Scale the number of nodes in the cluster.
 func (h *Handler) Scale(ctx context.Context, request *grpc_provisioner_go.ScaleClusterRequest) (*grpc_infrastructure_manager_go.ProvisionerResponse, error) {
-	log.Debug().Interface("request", request).Msg("Scale cluster")
+	log.Debug().Str("request", request.OrganizationId).
```
Comment (rodynnz) [inline internal/pkg/server/clusters/handler.go]: We pass request with the OrganizationId and organizationID with also OrganizationId? Isn't that redundant?

File: internal/pkg/server/clusters/handler.go [hunk 2]
```diff
@@ -88,6 +90,54 @@ func (h *Handler) Scale(ctx context.Context, request *grpc_provisioner_go.ScaleC
 	return h.Manager.Scale(request)
 }
 
+// Uninstall a existing cluster. This process will uninstall the nalej platform and
+// remove the cluster from the list.
+func (h *Handler) Uninstall(ctx context.Context, request *grpc_public_api_go.UninstallClusterRequest) (*grpc_public_api_go.OpResponse, error) {
+	log.Debug().Str("request", request.OrganizationId).
```
Comment (rodynnz) [inline internal/pkg/server/clusters/handler.go]: Same question as above

File: internal/pkg/server/clusters/handler.go [hunk 3]
```diff
@@ -88,6 +90,54 @@ func (h *Handler) Scale(ctx context.Context, request *grpc_provisioner_go.ScaleC
...
+		return nil, opErr
+	}
+	return entities.ToPublicAPIOpResponse(response), nil
+}
+
+// Decomission an application cluster. This process will uninstall the nalej platform,
+// decomission the cluster from the infrastructure provider, and remove the cluster from the list.
+func (h *Handler) Decomission(ctx  context.Context, request *grpc_public_api_go.DecomissionClusterRequest) (*grpc_public_api_go.OpResponse, error) {
+	log.Debug().Str("request", request.OrganizationId).
+		Str("organizationID", request.OrganizationId).Str("clusterID", request.ClusterId).
```
Comment (rodynnz) [inline internal/pkg/server/clusters/handler.go]: Same as above

File: internal/pkg/server/clusters/handler.go (hunk 1 above)
Comment (dhiguero) [reply internal/pkg/server/clusters/handler.go]: I think there is a copy&paste typo that is creating some confussion`request.OrganizationId` is only a value, not a message. I will change the message.

**Pull request merged**